changelog.d/14565.misc

```diff
@@ -0,0 +1 @@
+In application service transactions that include the experimental `org.matrix.msc3202.device_one_time_key_counts` key, include a duplicate key of `org.matrix.msc3202.device_one_time_keys_count` to match the name proposed by [MSC3202](https://github.com/matrix-org/matrix-spec-proposals/blob/travis/msc/otk-dl-appservice/proposals/3202-encrypted-appservices.md).
```

Comment:
when linking to MSCs, please link via the pull request URL - the markdown URL is not static and subject to change.

Reply:
Thanks -- I'll keep this in mind in the future.
I can make a new PR to fix this changelog, if it's worth the effort.